app/controllers/reporting_controller.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from flask import current_app, abort
+
+
+def get_reporting_details(collectioninstrumenttype, collex_id):
+
+    url = f'{current_app.config["REPORTING_URL"]}reporting-api/v1/response-dashboard' \
+          f'/{collectioninstrumenttype}/collection-exercise/{collex_id}'
+    try:
```

Comment:
I think you should check the validity of the arguments and fail early, instead of making a call to `rm_reporting` then failing.